drivers/i2c/l3gd20h_driver.go

```diff
@@ -0,0 +1,135 @@
+package i2c
+
+import (
+	"bytes"
+	"encoding/binary"
+	
+	"gobot.io/x/gobot"
+)
+
+const l3gd20hAddress = 0x6B
+
+// Control Register 1
+const l3gd20hRegisterCtl1 = 0x20
+const l3gd20hNormalMode = 0x8
+const l3gd20hEnableZ = 0x04
+const l3gd20hEnableY = 0x02
+const l3gd20hEnableX = 0x01
+
+// Control Register 4
+const l3gd20hRegisterCtl4 = 0x23
+
+const l3gd20hRegisterOutXLSB = 0x28 | 0x80 // set auto-increment bit.
+
+// L3GD20HDriver is the gobot driver for the Adafruit Triple-Axis Gyroscope L3GD20H.
+// Device datasheet: http://www.st.com/internet/com/TECHNICAL_RESOURCES/TECHNICAL_LITERATURE/DATASHEET/DM00036465.pdf
+type L3GD20HDriver struct {
+	name       string
+	connection I2c
+	scale L3GD20HScale
+}
+
+// L3GD20HScale is the scale sensitivity of degrees-per-second.
+type L3GD20HScale byte
+
+const (
+	// L3GD20HScale250dps is the 250 degress-per-second scale.
+	L3GD20HScale250dps = 0x00
+	// L3GD20HScale500dps is the 500 degress-per-second scale.
+	L3GD20HScale500dps = 0x10
+	// L3GD20HScale2000dps is the 2000 degress-per-second scale.
+	L3GD20HScale2000dps = 0x30
+)
+
+// NewL3GD20HDriver creates a new driver with the i2c interface for the L3GD20H device.
+func NewL3GD20HDriver(c I2c, s L3GD20HScale) *L3GD20HDriver {
+	return &L3GD20HDriver{
+		name:       "L3GD20H",
+		connection: c,
+		scale: s,
```

Comment:
I suggest instead of requiring a param for scale, instead set a sensible default in line 49, and then add a `SetScale()` method. What do you think?